app/controllers/journeys/expenses/tailoring/ExpensesTailoringCYAController.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.journeys.expenses.tailoring
+
+import controllers.actions._
+import controllers.journeys.expenses.tailoring
+import models.common.{BusinessId, TaxYear}
+import pages.expenses.tailoring.ExpensesTailoringCYAPage
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
+import viewmodels.journeys.SummaryListCYA
+import views.html.journeys.expenses.tailoring.ExpensesTailoringCYAView
+
+import javax.inject.Inject
+import scala.annotation.nowarn
+
+class ExpensesTailoringCYAController @Inject() (
+    override val messagesApi: MessagesApi,
+    identify: IdentifierAction,
+    getData: DataRetrievalAction,
+    requireData: DataRequiredAction,
+    val controllerComponents: MessagesControllerComponents,
+    view: ExpensesTailoringCYAView
+) extends FrontendBaseController
+    with I18nSupport {
+
+  def onPageLoad(taxYear: TaxYear, businessId: BusinessId): Action[AnyContent] = (identify andThen getData andThen requireData) { implicit request =>
+    val userType    = request.userType
+    val summaryList = SummaryListCYA.summaryListOpt(Nil) // TODO Add during Expenses tailoring CYA story
```

Comment:
Is there a ticket for this anywhere? I can't see one, but would be good to put the JIRA reference here if so?

Reply:
not yet, waiting for Chris to create one